math/mean_absolute_error.cpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file mean_absoulte_eror.cpp
+ * @brief Program ini merupakan untuk
+ * menghitung Mean Absolute Error dari 
+ * 2 array dengan panjang yang sama
+*/
+#include <iostream>
+#include <vector>
+#include <exception>
+#include <cmath>
+/**
+ * @brief Fungsi mae untuk menghitung mean Absolute Error
+ * yang bertujuan mengetahui berapa akurasi error pada model
+ * data yang kita latih
+ * 
+*/
+double mae(const std::vector<double> &predict,const std::vector<double>&actual){
+    if (predict.size() != actual.size()){
+        std::invalid_argument("panjang datanya tidak sama\ntolong check lagi");
+    }
+    double abs_sum=0.0;
+    for(int i=0;i<predict.size();i++){
+        double error_value=(actual[i] - predict[i]);
+        error_value=std::abs (error_value);
+        abs_sum+=error_value;
```

Comment:
```suggestion
double error_value = std::abs(actual[i] - predict[i]);
        abs_sum += error_value;
```
bisa lebih diserderhanakan lagi @kayabaakihiko13